assignments/callbacks.js

```diff
@@ -39,27 +39,61 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 */
 
 
-function getLength(arr, cb) {
+function getLength(items, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(items.length);
 }
+ const cb = (items) => {
+    return items;
+};
 
-function last(arr, cb) {
+console.log(getLength(items, cb));
+
+function last(items, cb) {
   // last passes the last item of the array into the callback.
+  return cb(items[items.length - 1])
 }
 
+const callback = items => {
+   return items
+};
+
+console.log(last(items, cb));
+
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x,y);
+}
+
+const add = (x,y) => {
+  return x + y;
 }
 
+console.log(sumNums(1, 2, add));
+
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x, y,cb);
```

Comment:
Right here, you don't need to pass `cb` into the `cb()` function. You can just have
```
return cb(x, y)
```